CHANGELOG.md

```diff
@@ -39,6 +39,7 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 - Return the existing instrument when a view transforms a creation call to match an existing instrument. (#3240, #3251)
 - Log a warning when a conflicting instrument (e.g. description, unit, data-type) is created instead of returning an error. (#3251)
 - The OpenCensus bridge no longer sends empty batches of metrics. (#3263)
+- `sdktrace.TraceProvider.Shutdown` and `sdktrace.TraceProvider.ForceFlush` to not return error when no processor register. (#3268)
```

Comment:
This needs to be moved to the unreleased section.

Reply:
FYI: This is blocking collector on some things....